TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -66,7 +66,7 @@ repositories {
 dependencies {
     implementation("org.dafny:DafnyRuntime:${dafnyVersion}")
     implementation("software.amazon.smithy.dafny:conversion:0.1")
-    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.5.0")
+    implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.5.0-SNAPSHOT")
```

Comment:
This points the TestVectors to the local HEAD, as compared to the published result from Maven.
Outside of a release, TestVectors should point to HEAD, so that HEAD is tested.

During the last release, someone forgot to restore this.